### ISSUE TITLE
PHP Fatal error when trying to export label to MyParcel

### ISSUE THREAD

**Issue by JanMiddelkoop** [myparcelnl/prestashop#171]
### Module version

1.6.1

### PrestaShop version

1.7.8.6

### PHP version

7.4

### What went wrong?

When exporting addresses to MyParcel, my client gets a Server Error and in the logs is a PHP Fatal Error.

### Reproduction steps

1. In the backoffice navigate to "Orders"
2. Click on "Create" for an order
3. Choose label amount 1, package type digital stamp, weight 50 - 100g
4. Click "Exporteren"

### Relevant log output

```shell
[2022-06-20 15:46:09] request.INFO: Matched route "myparcelnl_order". {"route":"myparcelnl_order","route_parameters":{"_controller":"Gett\\MyparcelNL\\Controllers\\Admin\\AdminMyParcelOrderController::index","_route":"myparcelnl_order"},"request_uri":".../admin/index.php/modules/myparcelnl/order?_token=Ytf212NrsxNezASQD_tVB2YOoh5PhfTCVk6RMZuXyek&action=export&orderIds=16797","method":"POST"} []
[2022-06-20 15:46:09] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-06-20 15:46:09] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"info@..."} []
[2022-06-20 15:46:09] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Type error: Argument 1 passed to MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment::setPerson() must be of the type string, null given, called in .../modules/myparcelnl/src/Factory/Consignment/ConsignmentFactory.php on line 242" at .../modules/myparcelnl/vendor/myparcelnl/sdk/src/Model/Consignment/AbstractConsignment.php line 1158 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Type error: Argument 1 passed to MyParcelNL\\Sdk\\src\\Model\\Consignment\\AbstractConsignment::setPerson() must be of the type string, null given, called in .../modules/myparcelnl/src/Factory/Consignment/ConsignmentFactory.php on line 242 at .../modules/myparcelnl/vendor/myparcelnl/sdk/src/Model/Consignment/AbstractConsignment.php:1158)"} []
[2022-06-20 15:46:09] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
```

**Comment by rockincent** [comment]
Same here

Module version
1.6.1

PrestaShop version
1.7.8.6

PHP version
7.4

**Comment by RichardPerdaan** [comment]
Fixed in [v1.7.0](https://github.com/myparcelnl/prestashop/releases/tag/v1.7.0)
